fixtures/large/rspec_mocks_proxy_expected.rb

```diff
@@ -414,25 +416,23 @@ def original_method_handle_for(message)
 
       def original_unbound_method_handle_from_ancestor_for(message)
         double = method_double_from_ancestor_for(message)
-        double && double.original_method.unbind
```

Comment:
it is correct for standard to make this change, it's not a change rubyfmt can make